Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -183,6 +189,103 @@ def get_asset_details_command(client: Client, args: dict) -> CommandResults:
     )
 
 
+def parse_expiration_date(expiration: Optional[str]) -> Optional[Union[int, str]]:
+    """
+     Converts relative expiration strings / numbers to epoch milliseconds or returns 'Never'.
+
+    Args:
+        expiration Optional[str]: The input from the command argument
+
+    Returns:
+        Optional[int, str]: The valiue that represent the expiration date of the IOC rule:
+            None: the rule get a default value.
+            str: "Never" - The rule has no expiration date.
+            int: epoch milliseconds of the expiration date.
+    """
+
+    if not expiration:
+        return None
+    if expiration == "Never":
+        return "Never"
+
+    def convert_datetime_to_epoch_milli(dt: datetime) -> int:
+        return int(dt.timestamp() * 1000)
+
+    def is_relative_time_format(s: str) -> bool:
+        """
+        Returns True if the input matches a relative time format:
+        'N minutes', 'N hours', 'N days', 'N weeks', 'N months', 'N years'
+        """
+        if not isinstance(s, str):
+            return False
+
+        pattern = r"^\s*\d+\s+(minutes|hours|days|weeks|months|years)\s*$"
+        return bool(re.match(pattern, s, flags=re.IGNORECASE))
+
+    now_epoch_milli = convert_datetime_to_epoch_milli(get_current_time())
+    try:
+        datetime_arg = arg_to_datetime(expiration)
+    except ValueError:
+        return expiration
+
+    if datetime_arg:
+        is_relative_time_format_flag = is_relative_time_format(expiration)
+        datetime_arg_epoch_milli = convert_datetime_to_epoch_milli(datetime_arg)
+        if is_relative_time_format_flag:
+            # Using arg_to_datetime that takes relative time and converts it into datetime (if its relative then in the past)
+            delta = now_epoch_milli - datetime_arg_epoch_milli
+            # the arg_to_datetime returned a time in the past
+            return now_epoch_milli + delta
+        else:
+            return datetime_arg_epoch_milli
+    else:
+        raise DemistoException("The expiration date cannot be converted to epoch milliseconds.")
+
+
+def prepare_ioc_to_output(ioc_payload: Union[dict, str], input_format: str) -> dict:
+    """
+    Prepare the IOC data to output:
+        if it's a Dictionary - return it, else converts a single-row CSV IOC definition into a JSON object (Python dict).
+
+    Args:
+        ioc_payload Union[dict, str]: the data contained in the IOC payload.
+        input_format str: representing what is the input format.
+
+    Returns:
+        dict: Parsed JSON-style IOC object.
+    """
+    if input_format == "JSON":
```

Comment:
```suggestion
    if input_format.upper() == "JSON":
```
Better safe than sorry